addons/website_slides/models/slide_channel.py

```diff
@@ -531,10 +530,20 @@ def action_redirect_to_members(self, completed=False):
         """ Redirects to attendees of the course. If completed is True, a filter
         will be added in action that will display only attendees who have completed
         the course. """
-        action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
         action_ctx = {'active_test': False}
+        action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
         if completed:
             action_ctx['search_default_filter_completed'] = 1
+        action['domain'] = [('channel_id', 'in', self.ids)]
+
+        if completed:
+            help_message = [_("No Attendees has completed this course yet!"), ""]
+        else:
+            help_message = [
+            _("No Attendees Yet!"),
+            _("From here you'll be able to monitor attendees and to track their progress.")
+        ]
```

Comment:
Indentation issue (PEP)
```suggestion
            help_message = [
                _("No Attendees Yet!"),
                _("From here you'll be able to monitor attendees and to track their progress.")
            ]
```